src/python/nimbusml/pipeline.py

```diff
@@ -819,6 +819,12 @@ def _fit_graph(self, X, y, verbose, **params):
                     model=output_predictor_model)
                 output_predictor_model_node._implicit = True
                 graph_nodes.append(output_predictor_model_node)
+        elif learner_node == None and len(transform_models) == 1 and transform_models[0] == "$dataset_transformer_model0":
```

Comment:
Will this work if there are transformers before the DatasetTransformer?

Reply:
Good point! I just pushed a fix for this case, and added new test case to reflect this possibility.